### PR TITLE
List group memberships of user in admin interface

### PR DESCRIPTION
**Pull request by chdorner**
This just shows a table of groups the user belongs to underneath the user data in the admin interface. The whole block is omitted when the user is not a member of any group.

2016-03-24-105911_560x390_scrot

### CODE REVIEW

**Comment by chdorner** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;jXlzp8Iz&#x2F;301-in-the-admin-interface-add-groups-this-user-belongs-to-to-the-user-data"> In the admin interface, add &quot;groups this user belongs to&quot; to the user data</a>

**Comment by robertknight** [comment]
If the list of groups could be sorted by name (case-insensitive) that would be useful when the user is a member of many groups.

Also worth adding a note to the CHANGES file.

**Comment by chdorner** [comment]
@robertknight added the ordering. Will add the `CHANGES` on master after the merge, that file is always a source of confusion when PR's get merged.

File: h/groups/models.py [hunk 1]
```diff
@@ -34,9 +34,9 @@ class Group(Base, mixins.Timestamps):
     creator = sa.orm.relationship('User')
 
     # Group membership
-    members = sa.orm.relationship('User',
-                                  secondary='user_group',
-                                  backref='groups')
+    members = sa.orm.relationship(
+        'User', secondary='user_group', backref=sa.orm.backref(
+            'groups', order_by=sa.func.lower(name)))
```
Comment (nickstenning) [inline h/groups/models.py]: using a reference to `name` here seems a bit magical to me. I think `'Group.name'` would probably be a bit clearer.

Comment (nickstenning) [inline h/groups/models.py]: Although perhaps that doesn't work because you're lowercasing things first?

Comment (chdorner) [inline h/groups/models.py]: that's what I wanted to do first, but I'm getting this: `NameError: name 'Group' is not defined`

**Comment by robertknight** [comment]
LGTM

**Pull request merged**